### PR TITLE
internal<->public sync

### PR DESCRIPTION
**Pull request by 2mia**
/ping @ddascal @mstefane

### CODE REVIEW

**Comment by mstefane** [comment]
LGTM

**Review by mstefane — APPROVED** [review]

**Comment by andralungu** [comment]
Looks great 👍. The title could be a bit more suggestive.

File: Dockerfile [hunk 1]
```diff
@@ -33,40 +33,29 @@ RUN apk update \
...
-# COPY src /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION}/src
-# COPY tests /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION}/tests
-# COPY Makefile /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION}/Makefile
-# RUN cd /tmp/api-gateway-zmq-adaptor-* \
-#         && make test \
-#         && mkdir -p /usr/local/sbin \
-#         && PREFIX=/usr/local/sbin make install \
-#         && rm -rf /tmp/api-gateway-zmq-adaptor-*
+# --- DEV ---
+RUN apk update \
```
Comment (ddragosd) [inline Dockerfile:42]: I guess we could leave this uncommented on the premise that this sidecar process runs within the container anyway and this docker image is used only for development ? Even though another design is to deploy a side-container that runs alongside the api gateway container, communicating via [ipc](https://docs.docker.com/engine/reference/run/#ipc-settings---ipc) .

**Review by ddragosd — APPROVED** [review]
LGTM, although I didn't test it locally.

File: Dockerfile (hunk 1 above)
Comment (2mia) [reply Dockerfile:42]: Q: "this docker image is used only for development ?" 
A: not yet, but trying to make it modular

**Pull request merged**